cluster-scope/bundles/volsync/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/core/namespaces/volsync-system
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/replicationdestinations.volsync.backube
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/replicationsources.volsync.backube
+  - ../../base/rbac.authorization.k8s.io/clusterrolebindings/volsync-manager
+  - ../../base/rbac.authorization.k8s.io/clusterrolebindings/volsync-proxy
+  - ../../base/rbac.authorization.k8s.io/clusterroles/volsync-manager
+  - ../../base/rbac.authorization.k8s.io/clusterroles/volsync-metrics-reader
```

Comment:
is the scc being imported else where? (I don't see it)

Also what about the namespace, is that something that would go in the bundle (I would think so)?

Reply:
> is the scc being imported else where? (I don't see it)

Whoops, yeah, missed that.

> Also what about the namespace, is that something that would go in the bundle (I would think so)?

Yeah, added that while you were reviewing :)

Reply:
Updated.